scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -470,8 +470,10 @@ class Router(formatOps: FormatOps) {
           Split(NewlineT(isDouble = tok.hasBlankLine), 0)
         )
 
-      case tok @ FormatToken(_: T.RightParen, _: T.KwDef | _: T.Comment, _)
-          if leftOwner.is[Defn.ExtensionGroup] =>
+      case tok @ FormatToken(_: T.RightParen, _, _)
+          if leftOwner.is[Defn.ExtensionGroup] && nextNonComment(
```

Comment:
can you please manually break after && and format rest on one line?

Reply:
Fixed!